src/bridge/key_loader/key_loader.cc

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <utility>
+
+#include "src/backing/client/crypto_key_version_algorithm.h"
+#include "src/backing/crypto_key_handle/crypto_key_handle.h"
+#include "src/backing/status/status.h"
+#include "src/backing/status/status_or.h"
+#include "src/bridge/error/error.h"
+#include "src/bridge/ex_data_util/ex_data_util.h"
+#include "src/bridge/key_loader/rsa_key_loader.h"
+#include "src/bridge/memory_util/openssl_bio.h"
+#include "src/bridge/memory_util/openssl_structs.h"
+
+namespace kmsengine {
+namespace bridge {
+
+using ::kmsengine::backing::CryptoKeyHandle;
+using ::kmsengine::backing::PublicKey;
+
+EVP_PKEY *LoadCloudKmsKey(ENGINE *engine, const char *key_id,
+                          UI_METHOD */*ui_method*/, void */*callback_data*/) {
+  KMSENGINE_ASSIGN_OR_RETURN_WITH_OPENSSL_ERROR(
```

Comment:
This looks like a syntax error?

Reply:
The variable names are commented out because they're ignored variables (following the "unused parameter" style from  https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions). GitHub unfortunately highlights this incorrectly.

Reply:
Gotcha. The rest LGTM.